src/components/reports/ReportForm.test.tsx

```diff
@@ -139,21 +142,22 @@ jest.setTimeout(20000);
 
 describe("Report form", () => {
   beforeEach(() => {
+    expect(router);
```

Comment:
I think we can remove this, and then it's import (though haven't tested and may be missing some jest magic)